src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
-'use strict';
+"use strict";
 
-// write your code here
+const div = document.createElement("div");
+
+const success = (result) => {
+  div.className = "message";
+  div.innerText = result;
+  document.body.append(div);
+};
+
+const error = (result) => {
```

Comment:
The className is overwritten in each call of error or success function. If you want to add additional classes, use classList.add.